Base/res/ladybird/inspector.js

```diff
@@ -767,33 +767,63 @@ document.addEventListener("DOMContentLoaded", () => {
     document.addEventListener("keydown", event => {
         const UP_ARROW_KEYCODE = 38;
         const DOWN_ARROW_KEYCODE = 40;
+        const RIGHT_ARROW_KEYCODE = 39;
+        const LEFT_ARROW_KEYCODE = 37;
         const RETURN_KEYCODE = 13;
         const SPACE_KEYCODE = 32;
 
-        if (document.activeElement.tagName !== "INPUT") {
-            if (event.keyCode == UP_ARROW_KEYCODE || event.keyCode == DOWN_ARROW_KEYCODE) {
-                let selectedIndex = visibleDOMNodes.indexOf(selectedDOMNode);
-                if (selectedIndex < 0) {
-                    return;
-                }
+        const navigateUp = () => {
+            let selectedIndex = visibleDOMNodes.indexOf(selectedDOMNode);
+            if (selectedIndex < 0) {
+                return;
+            }
 
-                let newIndex;
+            let newIndex = selectedIndex - 1;
 
-                if (event.keyCode == UP_ARROW_KEYCODE) {
-                    newIndex = selectedIndex - 1;
-                } else if (event.keyCode == DOWN_ARROW_KEYCODE) {
-                    newIndex = selectedIndex + 1;
-                }
+            if (visibleDOMNodes[newIndex]) {
+                inspectDOMNode(visibleDOMNodes[newIndex]);
+            } 
+        }
 
-                if (visibleDOMNodes[newIndex]) {
-                    inspectDOMNode(visibleDOMNodes[newIndex]);
-                }
+        const navigateDown = () => {
+            let selectedIndex = visibleDOMNodes.indexOf(selectedDOMNode);
+            if (selectedIndex < 0) {
+                return;
+            }
+
+            let newIndex = selectedIndex + 1;   
+
+            if (visibleDOMNodes[newIndex]) {
+                inspectDOMNode(visibleDOMNodes[newIndex]);
+            }
+        }
```

Comment:
Is there a reason to split navigating up and down into separate functions? Could also be done as `navigate(delta)` where
```js
let newIndex = selectedIndex + delta;
```
 would allow you to move any number of spaces up or down.

Reply:
Solid recommendation, I'll make the change